tests/common/helpers/pfc_storm.py

```diff
@@ -93,11 +118,17 @@ def deploy_pfc_gen(self):
         Deploy the pfc generation file on the fanout
         """
         if self.peer_device.os in ('eos', 'sonic'):
+            src_pfc_gen_file = "common/helpers/{}".format(self.pfc_gen_file)
             self._create_pfc_gen()
```

Comment:
Is the old pfc_gen script still required for Mellanox-SONiC leaf-fanout?

Reply:
Yes, because the lower level sdk api is based on Mellanox chip.